TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -48,6 +48,15 @@ const BenchmarkMetricRegex = {
                 regexRepeat: false
             }
         ]
+    },
+    AcmeAirNodejs: {
+        metrics: [
+            {
+                name: "Throughput",
+                regex: /Throughput: (\d+.\d+)/, //Example: "Throughput: 3554.1"
```

Comment:
Could you please change the `regex` to `/Throughput: (\d*\.?\d*)/` since it's safer? Current regex `/Throughput: (\d+.\d+)/` won't catch 2 digit numbers such as 89 since it expects 3 digits at least.

Reply:
Sure